ui/app/templates/components/job-page/parts/task-groups.hbs

```diff
@@ -40,7 +40,6 @@
         <TaskGroupRow
           @data-test-task-group={{row.model.name}}
           @taskGroup={{row.model}}
-          @onClick={{fn this.gotoTaskGroup row.model}}
```

Comment:
Side-effect: noticed in testing that clicking the exact centre of task group row's scaling buttons would route you into that task group. Long-term this gets fixed by upgrading to Helios tables.